1365.py

```diff
@@ -4,13 +4,14 @@ def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
         ans = []
         
         for num in nums:
-            count = 0
+            idx = sorted_nums.index(num)
             
-            for sorted_num in sorted_nums:
-                if num > sorted_num:
-                    count += 1
-            
-            ans.append(count)
-            count = 0
-        
+            while idx > 0:
```

Comment:
As specified in in the [docs](https://docs.python.org/3/tutorial/datastructures.html), `index()` returns you a `zero-based index in the list of the first item whose value is equal to x.` Therefore, there is no need to have this `while` loop shift your `idx` to the left any further. 

However, note that the implementation of `index()` makes no assumption on the list (or array) on whether or not it is already sorted, therefore it uses a linear searching algorithm to find the left-most position for you. This is the key difference between `list.index()` and `bisect.bisect_left()`.

Reply:
I need to give credit where credit is due: your `while` loop is correct, and it won't cause a `list index out of range` error. My point is that this `while` loop is trying to obtain an outcome that is already guaranteed by the `index()` function.